### DISCUSSION TITLE
Support for Kind

### DISCUSSION THREAD

**Discussion by roopeshsn** [microsoft/retina#555] [Q&A]
I noticed that this issue https://github.com/microsoft/retina/issues/111 was closed. Is Kind supported now? If yes then it should be updated in documentation.

**Comment by timraymond** [comment]
Hi @roopeshsn, thanks for bringing that to our attention. I've been able to run Retina in Kind (under WSL) for the past few months. I'll update the docs with my setup. Are you having issues with running Retina in Kind that we should know about?
